openedx/core/djangoapps/content/course_overviews/migrations/0026_courseoverview_entrance_exam.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.12 on 2022-02-25 20:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_overviews', '0025_auto_20210702_1602'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='courseoverview',
+            name='entrance_exam_enabled',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AddField(
+            model_name='courseoverview',
+            name='entrance_exam_id',
+            field=models.CharField(blank=True, max_length=255),
+        ),
+        migrations.AddField(
+            model_name='courseoverview',
+            name='entrance_exam_minimum_score_pct',
+            field=models.FloatField(default=0.65),
+        ),
+        migrations.AddField(
```

Comment:
I'm not sure I understand what historicalcourseoverview is. What do these fields do?

Reply:
The historical model keeps a changelog of all updates to the actual model. So say you update the description, the actual model will just update to the new description, but the historical model will have a row containing the old description and a row with the new description. Essentially, the historical model.latest() == model.get() if that makes sense.